crates/cargo-platform/tests/test_check_cfg.rs

```diff
@@ -0,0 +1,79 @@
+use cargo_platform::{CheckCfg, ExpectedValues};
+use std::collections::HashSet;
+
+#[test]
+fn check_cfg_none() {
+    let mut check_cfg = CheckCfg::default();
+
+    check_cfg.process_line("cfg_a");
+    assert_eq!(
+        *check_cfg.expecteds.get("cfg_a").unwrap(),
+        ExpectedValues::Some(HashSet::from([None]))
+    );
+}
+
+#[test]
+fn check_cfg_empty() {
+    let mut check_cfg = CheckCfg::default();
+
+    check_cfg.process_line("cfg_b=");
+    assert_eq!(
+        *check_cfg.expecteds.get("cfg_b").unwrap(),
+        ExpectedValues::Some(HashSet::from([]))
+    );
+}
+
+#[test]
+fn check_cfg_any() {
+    let mut check_cfg = CheckCfg::default();
+
+    check_cfg.process_line("cfg_c=any()");
+    assert_eq!(
+        *check_cfg.expecteds.get("cfg_c").unwrap(),
+        ExpectedValues::Any
+    );
+
+    check_cfg.process_line("cfg_c");
+    assert_eq!(
+        *check_cfg.expecteds.get("cfg_c").unwrap(),
+        ExpectedValues::Any
+    );
```

Comment:
Does `check-cfg` not allow you to specify that a `--cfg` can be either `name` or `name=value`?